app/views/admin/loans/content/_breakeven_table_report.html.slim

```diff
@@ -19,12 +19,15 @@
         tr.group-content
           td = item[:name]
           td.text-right = item[:quantity]
-          td.text-right = number_to_currency(item[:amount])
-          td.text-right = number_to_currency(item[:total])
+          td.text-right 
+            = @loan.ensure_currency.format_amount(item[:amount], tooltip: false)
```

Comment:
You're right, but the presenter approach seems a bit heavy. I'm not sure I agree that having all the helpers in the same namespace is such a big problem as to warrant that much extra code. Convention over configuration, right? I like your second choice.